quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/storage/database/liquibase/FastServiceLocator.java

```diff
@@ -79,6 +79,15 @@ public FastServiceLocator(Map<String, List<String>> services) {
         getPackages().remove("liquibase.parser.core.json");
```

Comment:
just a merely related note: should we make this class final to avoid extension of it? When someone would extend the class, all the overridable methods in the constructor could cause serious headache ;)

Reply:
And some others such as KeycloakMain. Feel free to open a PR and change this one or any other we should make final.

Reply:
will do :)